src/KristofferStrube.Blazor.FileSystemAccess/Options/DirectoryPickerOptions.cs

```diff
@@ -53,6 +54,13 @@ internal virtual ExpandoObject Serializable()
             res.id = Id;
         }
 
+        res.mode = Mode switch
```

Comment:
The `FileSystemPermissionMode` already has a JSON converter registered on the type so you don't need to map it to a string. You can set it directly and it will automatically be serialized as the correct string. :)

Reply:
Wow that's cool, I didn't know that's a thing. Learned something new today, thanks.